web/src/views/events/EventView.tsx

```diff
@@ -206,9 +206,9 @@ export default function EventView({
 
   return (
     <div className="flex flex-col size-full">
-      <div className="h-10 relative flex justify-between items-center mx-2">
+      <div className="h-11 px-2 relative flex justify-between items-center bg-red-500">
```

Comment:
roses are red, violets are blue, I love UI tweaks, and red divs too

Reply:
left that in just for you